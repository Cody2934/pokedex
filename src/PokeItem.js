import React, { Component } from 'react'

export default class PokeItem extends Component {
    render() {
        return ( 
        
        <div className="thePoke">

            {/* Pokemon Name */}
            {/* <h2 className="pokeName">{this.props.pokemon.name}</h2> */}

            {/* Pokemon Image */}
            <img className="pokePics" src={this.props.pokemon.url_image}/>

            {/* Pokemon HP */}
            <p className="pokeInfo">HP: {this.props.pokemon.hp}</p>

            {/* Pokemon Attack */}
            <p className="pokeInfo">Attack: {this.props.pokemon.attack}</p>

            {/* Pokemon Defense */}
            <p className="pokeInfo">Defense: {this.props.pokemon.defense}</p>
            
            {/* Pokemon Speed */}
            <p className="pokeInfo">Speed: {this.props.pokemon.speed}</p>

            {/* Pokemon Type */}
            <p className="pokeInfo">Type: {this.props.pokemon.type_1}</p>

        </div>
        
        )
    }
}