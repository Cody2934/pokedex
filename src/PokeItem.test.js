import React from 'react';
import PokeItem from './PokeItem.js';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<PokeItem pokemon={[]}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});