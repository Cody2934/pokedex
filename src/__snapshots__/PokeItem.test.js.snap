// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="thePoke"
>
  <img
    className="pokePics"
  />
  <p
    className="pokeInfo"
  >
    HP: 
  </p>
  <p
    className="pokeInfo"
  >
    Attack: 
  </p>
  <p
    className="pokeInfo"
  >
    Defense: 
  </p>
  <p
    className="pokeInfo"
  >
    Speed: 
  </p>
  <p
    className="pokeInfo"
  >
    Type: 
  </p>
</div>
`;
